pkg/controller/podautoscaler/OWNERS

```diff
@@ -1,14 +1,17 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
+  - adrianmoisey
   - gjtempleton
-  - mwielgus
+  - omerap12
+  - raywainman
```

Comment:
As I've stated in the linked issue, the maintainers need to earn their respective status. For the above 3 reviewers can you link me here [the required number of PRs](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-1) so that I can confirm they are eligible.

Reply:
discussed: https://github.com/kubernetes/kubernetes/pull/131650